frontend/packages/dev-console/src/test/pipeline-data.ts

```diff
@@ -1266,12 +1395,12 @@ export const pipelineTestData: PipelineTestData = {
           timeout: '1h0m0s',
         },
         status: {
+          pipelineSpec: pipelineSpec[PipelineExampleNames.INVALID_PIPELINE_MISSING_TASK],
           completionTime: '2020-07-13T17:21:05Z',
           conditions: [
             {
               lastTransitionTime: '2020-07-13T17:21:05Z',
-              message:
-                'Pipeline andrew/task-ref-error can\'t be Run; it contains Tasks that don\'t exist: Couldn\'t retrieve Task "not-a-task": task.tekton.dev "not-a-task" not found',
+              message: '',
```

Comment:
Any reason you got rid of this data text? We could trim out my name for sure haha, it was log dumps when I created. We could make the message something like `Test error text`, but I think it shouldn't be empty... as our tests could send that data in and it be mistaken as no error text rather than just an empty error text for testing.

Reply:
While consolidating the test data, there was a test case, where it returns a static message `Unknown failure condition` if the message field is emtpy. For that reason i added this. I have reverted back  the above one and handled the mock data in the test case itself.